Plots/Contours/NCL_polar_1_lg.py

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Tue May 12 09:38:46 2020
+
+@author: misi1684
+"""
+import numpy as np
```

Comment:
You may want to make use of another Sphinx feature here:

Before starting significant blocks within your code (e.g. pakage imports, data reading, or plotting lines), you can use following comment style:

```
###############################################################################
# Import packages:
```

to let Sphinx know to generate Jupyter notebook-like cells in our ReadTheDocs webpages. Example usages can be seen from previous functions.